src/app/log-in-page/log-in-page.component.css

```diff
@@ -0,0 +1,80 @@
+html {
+  height: 100%;
+}
+body
+
+.login-box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 400px;
+  padding: 40px;
+  transform: translate(-50%, -50%);
+  background: black;
+  box-sizing: border-box;
+  border-radius: 15px;
+}
+
+.login-box h2 {
+  margin: 0 0 30px;
+  color: white;
+  text-align: center;
+}
+
+.login-box .user-box {
+  position: relative;
+}
+
+.login-box .user-box input {
+  width: 100%;
+  padding: 10px 0;
+  font-size: 15px;
+  color: white;
+  margin-bottom: 30px;
+  border: black;
+  border-bottom: 1px solid white;
+  outline: black;
+  background: transparent;
+}
```

Comment:
this comment about code style. i think we need add empty string after all block style